wordpress-mobile/WordPress-iOS
# Aztec implement videos

This PR implements support for display videos in the Aztec editor.
The video can be VideoPress (WP.com) or regular WordPress videos (self-hosted).

This PR still **doesn't** implement insertion of video in Aztec, that will be address on a follow up PR.

To test:
 - Open the app
 - Go to blog -> Posts
 - Open a post that already has a video inserted and see if you can see the poster images (only works for videoPress or regular WordPress videos that have defined a poster image)
 - See if you tap on the video you can see the video playing in full screen.

Needs review: @jleandroperez do you mind have a look on this?
